components/Pages/CarePackages/ReviewPackageDetails/index.js

```diff
@@ -137,8 +136,6 @@ const ReviewPackageDetails = ({
     }));
   };
 
-  const isActivePackage = packageStatus < 6;
```

Comment:
Why remove this?

Reply:
i removed this change, it's not relevant to this branch